Jamulus.pro

```diff
@@ -109,6 +109,51 @@ win32 {
         INCLUDEPATH += "C:/Program Files (x86)/Jack/includes"
         LIBS += "C:/Program Files (x86)/Jack/lib/libjack64.lib"
     }
+} else:win64 {
```

Comment:
Is there an actual `win64` target?

Can't just use a `contains(QT_ARCH, x86_64)` conditional within the existing `win32` target? This should simplify the logic a lot and should save duplication if the sole intention is to change the jack path.

Reply:
@jujudusud , please note that the path for the jack include folder is now named include (without the s).
When I build the version locally with jackonwindows, I used the contains(QT_ARCH, x86_x64) which I believe saves duplicating/maintaining extra lines. And perhaps the message prompted that jack is not installed could change to "Warning: jack.h was not found at the usual place, maybe jack is not installed or it is not the latest version"

Thanks for doing this...I could have done it as well, but was waiting for feedback on the issue regarding the approach to take.

Reply:
Now using `win32` target and `contains(QT_ARCH, x86_64)` conditional.
Thanks for reviewing and helping.